assets/css/index.css

```diff
@@ -1,10 +1,44 @@
-#ProjectView {
-	width: 100%;
-
+body { 
+	height:100%;
+	background-color: rgba(125, 212, 250, 1);	
 }
 
 .row {
-	margin:4px;
-	padding: 8px;
 	min-height:40px;
+}
+
+#HistoryView {
+	margin-top: 50px; /* because of the header */ 
+	height:10%;
+	min-height: 60px;
+	padding:4px;
+	background-color: rgba(125, 212, 250, 1);
```

Comment:
We have a list of tacit colors: 

```
blue        2eabe2   
purple      807ec7
red         f15a5e
dk. brown   3d3130
md. brown   9c7b70
lt. brown   cdac9d
gray        d4d4d4
```

As used in the presentations. Perhaps we should use those?